Userland/Utilities/checksum.cpp

```diff
@@ -61,10 +65,68 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
             }
             file = file_or_error.release_value();
         }
+        if (!verify_from_paths) {
+            while (!file->eof() && !file->has_error())
+                hash.update(file->read(PAGE_SIZE));
+            outln("{:hex-dump}  {}", hash.digest().bytes(), path);
+        } else {
+            StringBuilder buffer;
+            while (!file->eof() && !file->has_error())
+                buffer.append(file->read(PAGE_SIZE));
+            Vector<StringView> lines = buffer.string_view().split_view("\n"sv);
+            if (lines.size() == 0) {
+                ++read_fail_count;
```

Comment:
It would be nice to have a unique warning for this situation, you can just print and continue here.